flatcar/scripts
# sys-kernel/bootengine: Install libcryptsetup-token-systemd-tpm2 plugin

This pulls in https://github.com/flatcar/bootengine/pull/93 to support systemd-cryptenroll for the rootfs with TPMs.


## How to use/Testing done

See above linked PR for Butane config.

Some notes:

- For better support we would need to teach Ignition to run systemd-cryptenroll. We might also need to add `systemd-tpm2-setup-early.service` and `{{LIBEXECDIR}}/systemd-tpm2-setup` for full support.
- I'll look into extending our qemu vm script template to allow setting up a swtpm more easily
- We need docs, also for the Clevis support once we bump the Butane Flatcar config version after enabling the clevis feature

- [x] Changelog entries added in the respective `changelog/` directory (user-facing change, bug fix, security fix, update)
- [x] Inspected CI output for image differences: `/boot` and `/usr` size, packages, list files for any missing binaries, kernel modules, config files, kernel modules, etc.
↓
```
Filesystem                 Size  Used Avail Use% Mounted on
/dev/vda1                  127M   62M   65M  49% /boot
```
